docs/_articles/en/getting-started/install.md

```diff
@@ -1,32 +1,33 @@
 ---
-title: Install Salesforce Code Analyzer Plugin
+title: Install Salesforce Code Analyzer
 lang: en
 ---
 
-## Install the plug-in
+## Install Salesforce Code Analyzer
```

Comment:
Since 2.x is being end-of-lifed, we should change this file to either recommend against installing it, or to indicate that it should be installed with a tag instead of latest.